src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.service.DataReaderService;
+import core.basesyntax.service.DataWriterService;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.ReportGeneratorService;
+import core.basesyntax.service.impl.CsvDataReaderServiceImpl;
+import core.basesyntax.service.impl.CsvDataWriterServiceImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.ReportGeneratorServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        final Map<String, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put("b", new BalanceOperationHandler());
+        operationHandlerMap.put("s", new SupplyOperationHandler());
+        operationHandlerMap.put("p", new PurchaseOperationHandler());
+        operationHandlerMap.put("r", new ReturnOperationHandler());
+
+        OperationStrategy strategy = new OperationStrategyImpl(operationHandlerMap);
+        DataReaderService dataReaderService = new CsvDataReaderServiceImpl();
+        ReportGeneratorService reportGeneratorService = new ReportGeneratorServiceImpl(strategy);
```

Comment:
ReportGeneratorService does not need a strategy - it should look at Storage and create report based on stock balance
```suggestion
        ReportGeneratorService reportGeneratorService = new ReportGeneratorServiceImpl();
```
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/1a78b3f6-e588-427c-9391-9a5eff577a8a)